source/Hero.hx

```diff
@@ -0,0 +1,22 @@
+package;
+
+import flixel.FlxSprite;
+
+class Hero extends FlxSprite
+{
+	// Implemented Constructor
+	public function new(x, y)
```

Comment:
Note that, since these aren't optional, every new Hero will *have* to have x & y coordinates passed in. That can be fine, it's just worth noting that it's different compared to the FlxSprite class that defaults the values to 0 if not provided.